server/email/config.js

```diff
@@ -1,8 +1,9 @@
 Meteor.startup(function() {
 
```

Comment:
Hi , I thought Mailgun config elements should put in settings-development( ||  settings- *******).json for some secure reasons.